app/views/course/levels/index.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.levels @levels.map(&:experience_points_threshold)
```

Comment:
Use `@levels.pluck(:experience_points_threshold)` instead - that will make the SQL only return the threshold rather than the entire object.

Reply:
It doesn't get sent to the frontend, rather it is just instantiated in the server during the API request (and is accessible through the views or json builder files). That said, `@levels` is still needed to do some authorisation, so maybe this is ok 👍